metricflow/model/validations/validator_helpers.py

```diff
@@ -286,6 +286,27 @@ def all_issues(self) -> Tuple[ValidationIssueType, ...]:
         """For when a singular list of issues is needed"""
         return self.fatals + self.errors + self.future_errors + self.warnings
 
+    def summary(self) -> str:
+        """Returns a stylized summary string for issues"""
+
+        fatals = click.style(
+            text=f"{ValidationIssueLevel.FATAL.name}S: {len(self.fatals)}",
```

Comment:
preference nit: I have a slight preference for putting the enum name manipulations on the enum class (not the whole thing, just that pluralization), so if someone changes the name for `FATAL` to `FATALS` you're at least less likely to end up with `FATALSS` here.

It's up to you, though.